packages/quantic/force-app/examples/main/lwc/exampleInsightPanel/resultTemplates/youtubeResultTemplate.html

```diff
@@ -1,6 +1,6 @@
 <template>
   <c-quantic-result-template is-any-preview-open={isAnyPreviewOpen} result-preview-should-not-be-accessible={resultPreviewShouldNotBeAccessible}>
-    <c-quantic-colored-result-badge slot="label" label={result.raw.objecttype}></c-quantic-colored-result-badge>
+    <c-quantic-colored-result-badge slot="label" label={result.raw.documenttype}></c-quantic-colored-result-badge>
```

Comment:
Why are we changing that in the example insight panel?

Reply:
This is not related to the tests added in this PR but this was causing an error in the display of the example insight panel, for youtube results, `objecttype` was not available, so an error was displayed. I updated the field we try to display to avoid this error.